.circleci/config.yml

```diff
@@ -108,6 +106,51 @@ jobs:
           paths:
             - release
 
+  windows:
+    executor:
+      name: win/default
+    working_directory: C:\Users\circleci\simplenote-electron
+    environment:
+      CSC_FOR_PULL_REQUEST: true
+    steps:
+      - checkout
+      - attach_workspace:
+          at: C:\Users\circleci\simplenote-electron
+      - run:
+          name: Install make
+          command: cinst make
+      - run:
+          name: Install node version
+          command: |
+            $NODE_VERSION = Get-Content .\calypso\.nvmrc
+            nvm install $NODE_VERSION
+            nvm use $NODE_VERSION
+      - run:
+          name: Npm install
+          command: |
+            # The release of nodejs currently being used by Calypso includes a buggy version 
+            # of node-gyp for Windows. This workaround unblocks Windows builds until Calypso node 
+            # includes node-gyp >= 6.0.1, at which point this snippet and node-gyp in simplenote-electron 
+            # devDependencies can be reverted.
+            # Ref: https://github.com/nodejs/node-gyp/issues/1933
+            npm config set node_gyp "$(Get-Location)\node_modules\node-gyp\bin\node-gyp.js"
+            npm ci
+      - run:
+          name: Build windows
+          command: |
+            # Workaround for Sign Tool "private key filter" bug in Circle's Windows image.
+            # Ref: https://travis-ci.community/t/codesigning-on-windows/
+            # 
+            # Fix: Import .p12 into the local certificate store. Sign Tool will use 
+            # package.json's `certificateSubjectName` to find the imported cert.
```

Comment:
You'll also need to add `certificateSubjectName` key to package.json (you can see how this was done in wp-desktop).

([ref](https://github.com/Automattic/wp-desktop/blob/d45d0d947b1d2d67b38580f9744197869f110e94/package.json#L150))

Reply:
https://github.com/Automattic/simplenote-electron/pull/1771/commits/fc7a2b225fcdc527e8755e2520e66db2d266e425